### PR TITLE
Feature: Improve start

### PR DESCRIPTION
**Pull request by Wolmin**
Clickup ticket: https://app.clickup.com/t/861mgjpbg

### CODE REVIEW

File: cmd/lukso/errors.go [hunk 1]
```diff
@@ -17,4 +17,7 @@ var (
 	errTransactionFailed       = errors.New("Transaction failed ")
 )
 
-const noSuchFlag = "no such flag"
+const (
+	noSuchFlag           = "no such flag"
+	folderNotInitialized = "Folder not initialized - please make sure that you are working in initialized directory. You can initialize the directory with the init command."
```
Comment (Hugoo) [inline cmd/lukso/errors.go]:
```suggestion
	folderNotInitialized = "Folder not initialized - please make sure that you are working in an initialized directory. You can initialize the directory with the init command."
```

File: cmd/lukso/main.go [hunk 2]
```diff
@@ -88,46 +88,52 @@ func main() {
...
+			HideHelpCommand: true,
 		},
 		{
-			Name:   "init",
-			Usage:  "Initializes the node working directory, it's structure, and network configuration",
-			Action: initializeDirectory,
-			Flags:  networkFlags,
-			Before: initializeFlags,
+			Name:            "init",
+			Usage:           "Initializes the node working directory, it's structure, and network configuration",
```
Comment (Hugoo) [inline cmd/lukso/main.go]:
```suggestion
			Usage:           "Initializes the node working directory, its structure, and network configuration",
```

File: cmd/lukso/main.go [hunk 3]
```diff
@@ -88,46 +88,52 @@ func main() {
...
+			HideHelpCommand: true,
 		},
 		{
-			Name:   "update",
-			Usage:  "Updates all or specific LUKSO node clients in the working directory to the newest version",
-			Action: updateClients,
-			Before: initializeFlags,
-			Flags:  updateFlags,
+			Name:            "update",
+			Usage:           "Updates all or specific LUKSO node clients in the working directory to the newest version",
```
Comment (Hugoo) [inline cmd/lukso/main.go]:
```suggestion
			Usage:           "Updates all or specific LUKSO node clients in the working directory to the latest version",
```

File: cmd/lukso/main.go [hunk 4]
```diff
@@ -88,46 +88,52 @@ func main() {
...
+			HideHelpCommand: true,
 			Subcommands: []*cli.Command{
 				{
-					Name:   "geth",
-					Usage:  "Updates your geth client to newest version",
-					Action: updateGethToSpec,
-					Flags:  gethUpdateFlags,
-					Before: initializeFlags,
+					Name:            "geth",
+					Usage:           "Updates your geth client to newest version",
```
Comment (Hugoo) [inline cmd/lukso/main.go]:
```suggestion
					Usage:           "Updates your geth client to latest version",
```

File: cmd/lukso/main.go [hunk 5]
```diff
@@ -88,46 +88,52 @@ func main() {
...
+					HideHelpCommand: true,
 				},
 				{
-					Name:   "prysm",
-					Usage:  "Updates your prysm client to newest version",
-					Action: updatePrysmToSpec,
-					Flags:  prysmUpdateFlags,
-					Before: initializeFlags,
+					Name:            "prysm",
+					Usage:           "Updates your prysm client to newest version",
```
Comment (Hugoo) [inline cmd/lukso/main.go]:
```suggestion
					Usage:           "Updates your prysm client to latest version",
```

File: cmd/lukso/main.go [hunk 6]
```diff
@@ -88,46 +88,52 @@ func main() {
...
+					HideHelpCommand: true,
 				},
 				{
-					Name:   "validator",
-					Usage:  "Updates your validator client to newest version",
-					Action: updateValidatorToSpec,
-					Flags:  validatorUpdateFlags,
-					Before: initializeFlags,
+					Name:            "validator",
+					Usage:           "Updates your validator client to newest version",
```
Comment (Hugoo) [inline cmd/lukso/main.go]:
```suggestion
					Usage:           "Updates your validator client to latest version",
```

File: cmd/lukso/start.go [hunk 7]
```diff
@@ -84,21 +83,28 @@ func (dependency *ClientDependency) Stop() error {
...
 		log.Errorf("Couldn't read from config file: %v", err)
 
 		return err
 	}
 
+	// TODO for now just check if installed - when multiple clients will be supported we can run it generically
+	executionClient := cfg.Execution()
+	consensusClient := cfg.Consensus()
+	if executionClient == "" || consensusClient == "" {
+		log.Error("No selected client found in config. Please make sure that you have installed your clients.")
```
Comment (Hugoo) [inline cmd/lukso/start.go]:
```suggestion
		log.Error("No selected client found in config file [cli-config.yml - maybe use var here]. Please make sure that you have installed your clients. You can use the install command to install clients.")
```

**Review by Hugoo — APPROVED** [review]
lgtm

Just, I think that "latest" works better than "newest" (like for release, docker images etc. the word "latest" is usually used.

File: cmd/lukso/start.go (hunk 7 above)
Comment (Wolmin) [reply cmd/lukso/start.go]: What about `No selected client found in LUKSO configuration file (...)`? To keep it sort of ambiguous from user? We don't want him to tinker with this file anyway, only CLI should be responsible for editing it.

Comment (Hugoo) [reply cmd/lukso/start.go]: ok

**Pull request merged**